nautobot/core/api/views.py

```diff
@@ -264,7 +264,7 @@ def get_queryset(self):
                     model_field = model._meta.get_field(field_instance.source)
                 except FieldDoesNotExist:
                     continue
-                if isinstance(model_field, (ManyToManyField, RelatedField, ManyToOneRel, TagsField)):
+                if isinstance(model_field, (ManyToManyField, ManyToManyRel, RelatedField, ManyToOneRel, TagsField)):
```

Comment:
I noticed when adding new tests using the `IPAddress` view that we weren't optimizing the `interfaces` and `vm_interfaces` reverse-m2m lookups. Whoops!